src/aat/resources/features/F-026/F-026.feature

```diff
@@ -0,0 +1,58 @@
+@F-026
+Feature: F-026: Get a case data with UI layout
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-078
+  Scenario: must return the list of cases and status code 200 for correct inputs
+    Given a case that has just been created as in [Standard_Full_Case_Creation_Data]
+    And a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [uses case-reference of the case just created]
+    And it is submitted to call the [Get a case data with UI layout] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [contains the details of the case just created, along with an HTTP-200 OK]
```

Comment:
For God's sake, where is the check for the case just created?

Reply:
Updated:   And the response [contains details of existing cases associated, along with an HTTP-200 OK]